src/resources/2015/ChateauMantegna.ts

```diff
@@ -18,6 +26,17 @@ export function fightPainting(): string {
   return runCombat();
 }
 
+export function rest(): void {
+  visitUrl("place.php?whichplace=chateau&action=chateau_restbox");
+}
+
+export function restIfFree(): boolean {
```

Comment:
This function should probably be taken out into a general library function, and could even take a callback that would dictate what rest it did